db/migrate/20171101102630_clean_up_duplicates_in_containers_tables.rb

```diff
@@ -0,0 +1,91 @@
+class CleanUpDuplicatesInContainersTables < ActiveRecord::Migration[5.0]
+  class ContainerBuild < ActiveRecord::Base; end
+  class ContainerBuildPod < ActiveRecord::Base; end
+  class ContainerGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+  class ContainerLimit < ActiveRecord::Base; end
+  class ContainerNode < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+  class ContainerProject < ActiveRecord::Base; end
+  class ContainerQuota < ActiveRecord::Base; end
+  class ContainerReplicator < ActiveRecord::Base; end
+  class ContainerRoute < ActiveRecord::Base; end
+  class ContainerService < ActiveRecord::Base; end
+  class ContainerTemplate < ActiveRecord::Base; end
+  class PersistentVolumeClaim < ActiveRecord::Base; end
+
+  class ContainerImage < ActiveRecord::Base; end
+  class ContainerImageRegistry < ActiveRecord::Base; end
+
+  class ContainerCondition < ActiveRecord::Base; end
+  class SecurityContext < ActiveRecord::Base; end
+  class ComputerSystem < ActiveRecord::Base; end
+  class ContainerEnvVar < ActiveRecord::Base; end
+  class ContainerLimitItem < ActiveRecord::Base; end
+  class ContainerPortConfig < ActiveRecord::Base; end
+  class ContainerQuotaScope < ActiveRecord::Base; end
+  class ContainerQuotaItem < ActiveRecord::Base; end
+  class ContainerServicePortConfig < ActiveRecord::Base; end
+  class ContainerTemplateParameter < ActiveRecord::Base; end
+  class ContainerVolume < ActiveRecord::Base; end
+  class CustomAttribute < ActiveRecord::Base; end
+  class Hardware < ActiveRecord::Base; end
+  class OperatingSystem < ActiveRecord::Base; end
+
+  class Container < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def duplicate_data_query_returning_min_id(model, unique_index_columns)
+    model.group(unique_index_columns).select("max(id)")
```

Comment:
method name has min but sql function is max?

Reply:
right, it was changed in last commit, so I need to also rename the function :-)